src/systems.rs

```diff
@@ -134,7 +134,8 @@ pub fn particle_spawner(
                 .z_value_override
                 .as_ref()
                 .map_or(0.0, |jittered_value| jittered_value.get_value(&mut rng));
-            let particle_scale = particle_system.scale.at_lifetime_pct(0.0);
+            let initial_scale = particle_system.initial_scale.get_value(&mut rng);
+            let particle_scale = initial_scale * particle_system.scale.at_lifetime_pct(0.0);
```

Comment:
Hmm. So this changes `scale` from an absolute scale to a relative, as we're always multiplying the scale value by the initial_scale. You commented to this effect on `initial_scale`, but maybe we should also call this out on the doc for `scale`, that it is relative to `initial_scale`.